### ISSUE TITLE
Make long diagnostic messages in the Dependencies tree easier to consume

### ISSUE THREAD

**Issue by drewnoakes** [dotnet/project-system#6056]
When a project or package reference has an error/warning, the message is shown in the Dependencies tree as a child item (modulo #3780 which will be fixed soon).

![image](https://user-images.githubusercontent.com/350947/78183648-fbc23c00-74b3-11ea-99c8-4bc86e8767e6.png)

Some diagnostics are very hard to read when presented this way, depending upon the format of the message produced by NuGet.

We should have a general way to make consuming these messages easier.

The error text is available in both the Error List:

![image](https://user-images.githubusercontent.com/350947/78183714-1b596480-74b4-11ea-817a-10c5a1a5dd7e.png)

And the browse object:

![image](https://user-images.githubusercontent.com/350947/78183752-2b714400-74b4-11ea-8730-1a0f13756a05.png)

Customer [feedback](https://developercommunity.visualstudio.com/content/problem/963841/fix-this-very-long-error-in-solution-explorer-tree.html) is that the Error List is not discoverable enough, and I expect the same is true of the properties panel.

Ideas:

- If the message is a single line of text, wrap the tree item's caption at a fixed number of characters.
- Add a context menu item to copy the text to the clipboard.
- Produce a richer tool tip (this will be possible once #5697 is complete)

**Comment by jjmew** [comment]
@drewnoakes I am inclined to put this under Backlog. Thoughts?

**Comment by drewnoakes** [comment]
@jjmew makes sense. We have plenty of higher priority stuff for 16.7 at the moment.
